stats/src/main/java/io/airlift/stats/cardinality/HyperLogLog.java

```diff
@@ -97,7 +99,7 @@ public long cardinality()
 
     public int estimatedInMemorySize()
```

Comment:
while you are at it, it would be nice to make these size-related methods to return `long`.